app/Platform/RouteServiceProvider.php

```diff
@@ -24,6 +26,13 @@ public function boot(): void
         Route::pattern('game', '[0-9]{1,17}');
         Route::pattern('game_hash', '[a-zA-Z0-9]{1,32}');
         Route::pattern('system', '[0-9]{1,17}');
+
+        /*
+         * Don't reference hash identifiers by their raw ID
+         */
+        Route::bind('gameMd5Hash', function ($value) {
+            return GameHash::where('MD5', $value)->firstOrFail();
+        });
```

Comment:
The idea is to migrate all "active" hashes to the `hash` field and use `md5` for the hash that's provided by external sources.
Maybe we could move into that direction by calling the route key `gameHash`?
```php
Route::bind('gameHash', function ($value) {
    return GameHash::where('MD5', $value)->firstOrFail();
});
```

Reply:
Understood, adjustment made in latest.